libs/react-renterd/src/worker.ts

```diff
@@ -55,6 +55,36 @@ export function useObjectUpload(
   )
 }
 
+export type MultipartUploadChunkParams = {
+  key: string
+  bucket?: string
+  contractset?: string
+  uploadid: string
+  partnumber: number
+  minshards?: number
+  totalshards?: number
+  offset?: number
+  disablepreshardingencryption?: boolean
```

Comment:
you can drop `disablepreshardingencryption` now too

Reply:
Do you mean this option is no longer available? These hooks should mostly match the entire API

Reply:
Cool updating it. Does this mean `offset` is always required then?